src/sentry/tasks/post_process.py

```diff
@@ -203,6 +205,11 @@ def post_process_group(
                 ):
                     safe_execute(callback, event, futures)
 
+            has_workflow_owners = features.has("projects:workflow-owners-ingestion", event.project)
+            if has_workflow_owners:
+                processed_suspect_commits = True
+                process_suspect_commits.delay(group_id=group_id, cache_key=cache_key)
```

Comment:
Wondering if it makes sense to entirely skip this step if the org has no commits. We could cache the result of the check for a long time (probably an hour), since people switching from no commits to commits is infrequent.